templates/bans/bancomplete.tpl

```diff
@@ -1 +1 @@
-﻿Banned {$ban->getTarget()|escape} for {$ban->getReason()|escape}
\ No newline at end of file
+﻿Banned set for {$ban->getReason()|escape}
```

Comment:
Language here seems both unclear and I am unsure where this is posted.

Reply:
:facepalm:

You know, I'm not actually sure where this is used either. A code search on the filename is coming up blank, so I don't think it actually *is* used. I suspect that's why I didn't notice this horrendous grammar choice when I was testing it

Reply:
It's not used; removed.